resources/blueprints/globals/social_media.yaml

```diff
@@ -155,6 +155,24 @@ sections:
                   field:
                     type: hidden
                     default: 'https://www.github.com/'
+            Pinterest:
+              display: Pinterest
+              fields:
+                -
+                  handle: url
```

Comment:
Can you change `url` to `handle`?

Reply:
Oh my I just saw I got the yaml messed up, will fix and sort

Reply:
Done. But it doesn't render correctly anymore. I can't see where I messed it up, can you @robdekort?

The handles aren't even saved it seems:
![image](https://user-images.githubusercontent.com/435825/139833793-60d188ef-1e5e-4b19-9a0c-ae73b6941d3e.png)

Reply:
Nevermind I've mixed old yaml of my current project with the new template. Should all be good here.

Reply:
Cool!